wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
+The storyboard generally should not exceed an [SB load](/wiki/SB_load) of `5.0x`.
+You can check out the storyboard by pressing `F2` or clicking the `Design` tab at the top of the screen in editor.
+
+The dimensions of the elements should not exceed the default size.
+
+### Preview Point
 
-Essentially, each point of your mod post should have a **"when"**, **"what"**, and **"why"**. When you do not have the "why" or the "what", the suggestions may be a little confusing for the mapper. This, in turn, makes your mod a lot less useful than it should be. Simply explaining your suggestions will make your mod a lot more useful.
+All maps must have a preview point.
+
+A preview point will be denoted by a yellow tick mark on the [timeline](/wiki/timeline).
+You can also check if a map has a preview point by going into [AiMod](/wiki/AiMod) and checking under the `Style` tab.
+
+The preview point must be the same in all difficulties.
+
+### Folder Organization
+
+To see the beatmapset folder, click `File` then `Open Song Folder`.
+
+The song's bit rate must be between 128KBps and 192 KBps.
+
+In some cases, an empty `.osb` file may exist.
+You must deleted these if more than one is present.
+
+Inspect the folder for any unused files within the [beatmapset](/wiki/beatmapset).
+However, if you see `thumb.db` files, you can ignore them since these are automatically created by your operating system.
+
+## Mod Post Mistakes
+
+When you make a mod post, it is very important to **not** do any of the following:
+
+- Under-explaining requested changes.
+  _When modding with an inexperienced mapper's map, it is important to explain why you want to change certain things._
+  - **Spacing**: This will help the new mapper grasp a better understanding of beatmapping.
+    On the other hand, more experienced mappers rarely make accidental spacing mistakes, so please explain **why** you think the spacing should be changed.
+  - **New/Remove Combo**: Many times adding new combos will help a map.
+    However, if the modder does not explain why they should make such a change, the mapper will not fully understand the reason behind doing so and may not change the combo.
+  - **1 grid to the _right_**: It is not always obvious why a hit object should be moved a grid space up, right, down, or left.
+    It is fine to suggest this, but try to explain why it should be moved.
+    A common reason for this is to make a pattern symmetrical.
+    - This sort of suggestion does not offer much to improve a beatmap.
+      Keep in mind your mod post should not only consist of such suggestions.
+  - **This shape is ugly**: Whether the shape is made of hit circles, sliders, or both; the mapper obviously will not understand what you are requesting.
+    What looks aesthetically pleasing to the mapper may not always look aesthetically pleasing to you.
+    If you are going to suggest as such, at least explain why and supply another suggestion that the mapper could use.
+- Offensive language
```

Comment:
We included it in the ranking criteria at one point because people apparently didn't know how to have civil discussions (what a surprise)